assets/components/paymentAmount/paymentAmount.scss

```diff
@@ -0,0 +1,16 @@
+.component-payment-amount {
+	border-radius: 1000px;
+	font-size: 24px;
+	color: #fff;
+	width: 60px;
+	height: 60px;
+	text-align: center;
+	background-color: gu-colour(neutral-1);
+	line-height: 60px;
+}
+
+.component-payment-amount--wide-value {
+	padding-left: 1em;
```

Comment:
Why are you using `em` here? Do you need a length that depends on the html's parent element?